ironfish/src/primitives/rawTransaction.test.slow.ts

```diff
@@ -0,0 +1,202 @@
+      value: 789n,
+    })
+    raw.burns.push({
+      assetId: TEST_ASSET_ID_2,
+      value: 5n,
+    })
+
+    const burnNoteA = new Note(
+      new NativeNote(
+        generateKey().publicAddress,
+        123456789n,
+        'some memo',
+        TEST_ASSET_ID_1,
+        account.publicAddress,
+      ).serialize(),
+    )
+
+    const burnNoteB = new Note(
+      new NativeNote(
+        generateKey().publicAddress,
+        123456789n,
+        'some memo',
+        TEST_ASSET_ID_2,
+        account.publicAddress,
+      ).serialize(),
+    )
+
+    raw.spends.push({ note: burnNoteA, witness })
+    raw.spends.push({ note: burnNoteB, witness })
+  }
+
+  if (options.withOutputs) {
+    const outputNote = new Note(
+      new NativeNote(
+        generateKey().publicAddress,
+        123456789n - raw.fee,
+        'some memo',
+        Asset.nativeId(),
+        account.publicAddress,
+      ).serialize(),
+    )
+
+    raw.outputs.push({ note: outputNote })
+  }
+
+  return raw
+}
+
+/**
+ * Given an array of possible flags from `TestRawTransactionOptions`, produces
+ * a sequence of all possible combinations of such flags.
+ *
+ * Example:
+ *
+ * ```
+ * > testOptionCombinations(['withExpiration', 'withFee'])
+ * [
+ *   { withExpiration: false, withFee: false },
+ *   { withExpiration: false, withFee: true  },
+ *   { withExpiration: true,  withFee: false },
+ *   { withExpiration: true,  withFee: true  },
+ * ]
+ * ```
+ */
+function testOptionCombinations(
+  flags: Readonly<Array<keyof TestRawTransactionOptions>>,
+): Array<Partial<TestRawTransactionOptions>> {
+  const combinations = []
+  for (let mask = 0; mask < 2 ** flags.length; mask++) {
+    const options: Partial<TestRawTransactionOptions> = {}
+    for (let index = 0; index < flags.length; index++) {
+      const flagName = flags[index]
+      const enabled = !!(mask & (1 << index))
+      options[flagName] = enabled
+    }
+    combinations.push(options)
+  }
+  return combinations
+}
+
+function describeTestOptions(options: Partial<TestRawTransactionOptions>): string {
+  const description = Object.entries(options)
+    .filter(([_flag, enabled]) => enabled)
+    .map(([flag, _enabled]) => flag)
+  if (description.length) {
+    return description.join(', ')
+  } else {
+    return 'empty'
+  }
+}
+
+describe('RawTransaction', () => {
+  const nodeTest = createNodeTest()
+
+  describe('postedSize', () => {
+    describe('v1', () => {
+      const flags = [
+        'withExpiration',
+        'withFee',
+        'withMints',
+        'withBurns',
+        'withOutputs',
+      ] as const
+
+      testOptionCombinations(flags).forEach((options) => {
+        // eslint-disable-next-line jest/valid-title
+        it(describeTestOptions(options), async () => {
+          const account = await useAccountFixture(nodeTest.wallet)
+
+          const raw = createTestRawTransaction(account, options)
+          const serialized = raw.post(account.spendingKey).serialize()
```

Comment:
What do you think of using a tx fixture? It makes the run time go from ~90s on my machine to ~12s, and on CI it looks like it takes almost 500 seconds: https://github.com/iron-fish/ironfish/actions/runs/5675255965/job/15380216885?pr=4116 `@ironfish/sdk:  PASS   @ironfish/sdk  src/primitives/rawTransaction.test.slow.ts (482.75 s)`

Kind of a goofy function for this use case, we could probably use something like a `usePostRawTxFixture` type function that literally just takes a raw tx and posts it, but this works at least for now

```suggestion
          const tx = await useTxFixture(
            nodeTest.wallet,
            account,
            account,
            () => {
              return raw.post(account.spendingKey)
            },
            undefined,
            undefined,
            false,
          )
          const serialized = tx.serialize()
```